app/uk/gov/hmrc/softdrinksindustrylevystub/services/Generator.scala

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.softdrinksindustrylevystub.services
+
+import cats.implicits._
+import org.scalacheck._
+import org.scalacheck.support.cats._
+import uk.gov.hmrc.smartstub.Enumerable.instances.utrEnum
+import uk.gov.hmrc.smartstub.{AdvGen, _}
```

Comment:
Change this import to only include wildcard rather than having a named parameter and wildcard